### PR TITLE
NO-JIRA: iptables-alerter streamlining

### PR DESCRIPTION
**Pull request by danwinship**
I'm not really sure why iptables-alerter is using a lot of CPU (or at least, why some people _think_ it's using a lot of CPU), but looking at what it does, the most likely CPU suck is `jq`. (Especially maybe if `crictl` returns A LOT of JSON for certain pods for some reason?)

So this fixes it to use golang template support instead, and also fixes it to make only a single call to `crictl` per pod. Also, while I was tracing the pod to try to figure out what was being slow, I realized there's really no point in checking if we already logged an event before even deciding if we want to log an event. (Running `iptables-save` is more work on the node itself, but running `kubectl` is probably more work for the cluster as a whole.)

### CODE REVIEW

**Comment by danwinship** [comment]
/assign @kyrtapz 

from #2451:
> as a future improvement we might consider storing the `crictl inspectp "${id}"` result in a variable to avoid multiple crio calls.

done! (but I felt like the simpler fix in 2451 was more backportable...)

File: bindata/network/iptables-alerter/002-script.yaml [hunk 1]
```diff
@@ -37,36 +24,31 @@ data:
...
+                     * as a go template (eg, to fill in {{.ReleaseVersion}} above), but
+                     * it also needs to contain a go template string (to pass to crictl
+                     * below). We do this by putting the crictl argument as a string
+                     * literal inside an argument of the outer template:
+                     *     ... --template {{`'BLAH BLAH BLAH'`}} ...
+                     * becomes
+                     *     ... --template 'BLAH BLAH BLAH' ...
+                     * in the actual ConfigMap.
+                     */
+                -}}
```
Comment (kyrtapz) [inline bindata/network/iptables-alerter/002-script.yaml:39]: The template doesn't work:
```
template: ../../bindata/network/iptables-alerter/002-script.yaml:29: comment ends before closing delimiter"
```
This should work:
```suggestion
                {{- /* 
                     * bindata/network/iptables-alerter/002-script.yaml is processed
                     * as a go template (eg, to fill in {{.ReleaseVersion}} above), but
                     * it also needs to contain a go template string (to pass to crictl
                     * below). We do this by putting the crictl argument as a string
                     * literal inside an argument of the outer template:
                     *     ... --template {{`'BLAH BLAH BLAH'`}} ...
                     * becomes
                     *     ... --template 'BLAH BLAH BLAH' ...
                     * in the actual ConfigMap.
                */ -}}
```

**Review by kyrtapz — COMMENTED** [review]
Looks good, there is just one issue with the template.

**Pull request merged**